omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -1,26 +1,31 @@
-OMERO.web walkthrough installation CentOS 7 and IcePy 3.6
-=========================================================
+OMERO.web deployed separately from OMERO.server on CentOS 7 and IcePy 3.6
+=========================================================================
 
+Please first read :doc:`../../server-centos7-ice36`.
 
-For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively source the following file::
+
+This is an example walkthrough for installing OMERO.web decoupled from the OMERO.server in a **virtual environment** using OMERO.py and a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively use the following ones::
     
     OMERO_USER=omero
     WEBPORT=80
     WEBSERVER_NAME=localhost
 
 
 
-Create local user omero, homedir :file:`/home/omero` (run as root)::
-    
-    if [ -z "$(getent passwd omero)" ]; then
-    
-        useradd -m omero
+**The following steps are run as root.**
+
+Create a local system user omero if required, create the homedir too :file:`/home/omero`::
     
-    fi
+    useradd -m omero
     
     chmod a+X /home/omero
 
-Install ZeroC IcePy 3.6. IcePy is managed by PyPI and will be installed as a part of OMERO.web requirements (run as root)::
+Installing prerequisites
+------------------------
+
+**The following steps are run as root.**
+
+Install ZeroC IcePy 3.6. IcePy is managed by `PyPI <https://pypi.python.org/pypi>`_, a package management system used to install and manage software packages written in Python. IcePy will be installed as a part of OMERO.web requirements::
```

Comment:
maybe, "... as part of the ..."

Reply:
Fixed in https://github.com/ome/omeroweb-install/pull/22